libdevcrypto/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(EXECUTABLE devcrypto)
 
 file(GLOB HEADERS "*.h")
 add_library(${EXECUTABLE} ${SRC_LIST} ${HEADERS})
-add_definitions(-D_GLIBCXX_USE_CXX11_ABI=1)
+#add_definitions(-D_GLIBCXX_USE_CXX11_ABI=1)
```

Comment:
Yeah - we certainly couldn't just take it out.     Would need some conditionals.